stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -149,6 +153,7 @@
             Map<String, Object> sourceTypeData,
             StripeSourceTypeModel sourceTypeModel,
             @SourceType String type,
+            String customType,
```

Comment:
Note: this is not a breaking change because `Source` has a package-private constructor.